website/docs/r/cen_instance_attachment.html.markdown

```diff
@@ -43,6 +43,7 @@ The following arguments are supported:
 * `instance_id` - (Required, ForceNew) The ID of the CEN.
 * `child_instance_id` - (Required, ForceNew) The ID of the child instance to attach.
 * `child_instance_region_id` - (Required, ForceNew) The region ID of the child instance to attach.
+* `child_instance_owner_id` - (Optional) The uid of the child instance. Only used when attach a child instance of other account.
```

Comment:
Please add a tag for `child_instance_owner_id` `- (Optional, Available in 1.42.0+ )`